### ISSUE TITLE
No Menu Button And no OSD

### ISSUE THREAD

**Issue by 1edenec** [CogentRedTester/mpv-file-browser#10]
Hello!
I don't have an MENU button on my laptop. I tried to change keybind on line 530 to another one:
`mp.add_key_binding('Ctrl+z','browse-files', toggle_browser)`

but when I'm trying to start it from mpv, I see:
```
[file_browser] Command osd-overlay: no argument hidden.
[file_browser] Command osd-overlay: no argument hidden.
```

and no any OSD appears.

my system:
Archlinux, bspwm, zsh, kitty term.

**Comment by CogentRedTester** [comment]
What version of mpv are you using? This script may only work on the latest couple of versions of mpv.

Additionally, instead of changing the code directly, you should put `Ctrl+z script-binding browse-files` into input.conf

**Comment by CogentRedTester** [comment]
Should be fixed as of 787bc525086fced0c75aba3f2ce6f18ac225adac on at least 0.32 & 0.31
